arch/arm/configs/cyanogen_flo_defconfig

```diff
@@ -30,10 +30,6 @@ CONFIG_NEED_MACH_IO_H=y
 CONFIG_NEED_MACH_MEMORY_H=y
 CONFIG_PHYS_OFFSET=0x80200000
 CONFIG_GENERIC_BUG=y
-CONFIG_GENERIC_TIME_VSYSCALL=y
```

Comment:
There are many changes, and some of the removes don't have an accompanying # ... is not set  lines, could you comment the commit description a bit on these changes you made to .config?